sbor-tests/tests/enum.rs

```diff
@@ -75,8 +75,52 @@ pub enum Mixed {
     I = 0b11011,
 }
 
+#[derive(Debug, PartialEq, Eq, Sbor)]
+enum FlattenEnum {
+    #[sbor(flatten)]
+    A {
+        #[sbor(skip)]
+        skipped: u32,
```

Comment:
Minor.
For more clarity I would use different types for `skipped` and `y` to clearly indicate, which one is skipped in `FlattenedEnum::A`.